support-frontend/assets/helpers/redux/checkout/product/selectors/isSupporterPlus.ts

```diff
@@ -20,37 +20,24 @@ export function isSupporterPlus(
 		return false;
 	}
 
-	const benefitsThreshold = getThresholdPrice(countryGroupId, contributionType);
-	const selectedAmount = selectedAmounts[contributionType];
-
-	if (selectedAmount === 'other') {
-		const otherAmount = otherAmounts[contributionType].amount;
-		return otherAmount ? parseInt(otherAmount) >= benefitsThreshold : false;
-	}
+	const thresholdPrice = getThresholdPrice(countryGroupId, contributionType);
+	const selectedAmount = getAmount(
+		selectedAmounts,
+		otherAmounts,
+		contributionType,
+	);
 
-	return selectedAmount >= benefitsThreshold;
+	return selectedAmount >= thresholdPrice;
```

Comment:
This looks good to me - but I wonder if we could have some time next week writing some tests to help ensure these methods are covering all the bases and we don't change something for a use case we didn't know about

Reply:
I did check the use cases but happy to learn how to write test harnesses next week in a new PR.